src/library/Uno.Themes.WinUI.Markup/Theme/PipsPager.cs

```diff
@@ -0,0 +1,247 @@
+						public static ResourceValue<Brush> Focused => new("PipsPagerSelectionIndicatorForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorForegroundSelected")]
+						public static ResourceValue<Brush> Selected => new("PipsPagerSelectionIndicatorForegroundSelected", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBackground")]
+						public static ResourceValue<Brush> Default => new("PipsPagerSelectionIndicatorBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("PipsPagerSelectionIndicatorBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("PipsPagerSelectionIndicatorBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBackgroundSelected")]
+						public static ResourceValue<Brush> Selected => new("PipsPagerSelectionIndicatorBackgroundSelected", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("PipsPagerSelectionIndicatorBackgroundDisabled", true);	
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBorderBrush")]
+						public static ResourceValue<Brush> Default => new("PipsPagerSelectionIndicatorBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("PipsPagerSelectionIndicatorBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBorderBrushPressed")]
+						public static ResourceValue<Brush> Pressed => new("PipsPagerSelectionIndicatorBorderBrushPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBorderBrushSelected")]
+						public static ResourceValue<Brush> Selected => new("PipsPagerSelectionIndicatorBorderBrushSelected", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("PipsPagerSelectionIndicatorBorderBrushDisabled", true);
+					}
+
+					public static class BorderThickness
+					{
+						[ResourceKeyDefinition(typeof(Thickness), "PipsPagerButtonBorderThickness")]
+						public static ResourceValue<Thickness> Default => new("PipsPagerButtonBorderThickness", true);
+					}
+
+					public static class Width
+					{
+						[ResourceKeyDefinition(typeof(double), "PipsPagerHorizontalOrientationButtonWidth")]
```

Comment:
Shouldn't it be HorizontalOrientation -> Width and not the other way around?

Reply:
These Width and Height classes should be removed and these values should be moved into the `Button` class that was mentioned in the previous comment.

`PipsPager.Resources.Button.HorizontalOrientationWidth`
`PipsPager.Resources.Button.HorizontalOrientationHeight`